llvm/lib/Support/Unix/Program.inc

```diff
@@ -71,6 +71,7 @@ ProcessInfo::ProcessInfo() : Pid(0), ReturnCode(0) {}
 ErrorOr<std::string> sys::findProgramByName(StringRef Name,
                                             ArrayRef<StringRef> Paths) {
   assert(!Name.empty() && "Must have a name!");
+#if !defined(__wasi__)
```

Comment:
Why doesn't this function work in wasi?

Although, actually, since process spawning doesn't even work in wasi, again, ISTM wasi should have its own llvm/lib/Support/WASI/Program.inc, which is just full of stub implementations, instead of ifdef'ing everything out here.